server/aesdsocket.c

```diff
@@ -1,242 +1,344 @@
 /* Author: Swapnil Ghonge
- * Assignment: Assignment 5 Part1 
+ * Assignment: Assignment 6 Part1 
```

Comment:
@swapnilghonge you should not check your `server/aesdsocket` binary into your repo